setup.py

```diff
@@ -6,7 +6,7 @@
     author = "John Emmons",
     author_email = "mail@johnemmons.com",
     url='https://github.com/jremmons/pyfakewebcam', 
-    license='GPLv3',
+    license='LGPLv3',
```

Comment:
You should use a classifier `License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)`; the `license` keyword is meant for values that don’t have a classifier.

Reply:
Some third-party tools like `pyp2rpm` are reading the `license` entry. Adding a classifier would be an additional change.